server/routers/users.js

```diff
@@ -65,14 +67,16 @@ router.get('/', async (req, res, next) =>
   {
     filters.push({ name: `%${name}%` });
   }
-
+  if(is_professor != null){
+    filters.push({is_professor:`%${is_professor}%`})
```

Comment:
`is_professor` is an integer, is it not? Why compare with a string?

Reply:
Actually, being in a query it is a string. You can check it by logging its type to the console.
![Screenshot 2021-01-17 150335](https://user-images.githubusercontent.com/53211959/104843815-442cd800-58d5-11eb-908c-d67681e0996c.png)
![Screenshot 2021-01-17 150349](https://user-images.githubusercontent.com/53211959/104843828-4b53e600-58d5-11eb-9c63-60369e453c33.png)

Reply:
Yeah, but you can parse it with `parseInt(...)`

Reply:
Yeah, but why should we complicate things?

Reply:
Because we compare a string to an int, which can lead to errors further down the line